scripts/makeOVPN.sh

```diff
@@ -269,15 +269,16 @@ echo "tls-auth Private Key found: $TA"
 } > "${NAME}${FILEEXT}"
 
 # Copy the .ovpn profile to the home directory for convenient remote access
-cp "/etc/openvpn/easy-rsa/pki/$NAME$FILEEXT" "/home/$INSTALL_USER/ovpns/$NAME$FILEEXT"
-chown "$INSTALL_USER" "/home/$INSTALL_USER/ovpns/$NAME$FILEEXT"
+INSTALL_HOME=$(cat /etc/passwd | grep "$INSTALL_USER" | cut -d: -f6)
+cp "/etc/openvpn/easy-rsa/pki/$NAME$FILEEXT" "$INSTALL_HOME/ovpns/$NAME$FILEEXT"
+chown "$INSTALL_HOME" "$INSTALL_HOME/ovpns/$NAME$FILEEXT"
 chmod o-r "/etc/openvpn/easy-rsa/pki/$NAME$FILEEXT"
```

Comment:
rather use numerical permission, these are more concrete, easier to read and understand, can you change this to `chmod 640` ?

Reply:
Is that really necessary?  My goal was just to get the proper install folder and use it, as opposed to assuming that every home folder resides in "/home"....

Reply:
Really necessary? Nope ... 
Does the code look better, more uniform and readable? definitely yes! 

is just instead of `chmod o-r` having `chmod 640` 

then all you have to do is to commit and push to xptsp:master ... Also ... this is being committed to the master branch and should be committed to test branch, nothing is pushed directly into master. 

could you kindly make that change so I can merge your pr? thanks a lot.

Reply:
Tis done....  Thanks!